xml/_filter.xml

```diff
@@ -83,6 +83,10 @@
       <typeFilter name="*" expose="false">
       </typeFilter>
     </namespaceFilter>
+    <namespaceFilter name="Microsoft.Azure.Cosmos.Serialization.HybridRow">
+      <typeFilter name="*" expose="false">
```

Comment:
Will it exclude sub-namespaces also?

Reply:
It _should_ based on the reading: https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/dotnet/filtering-apis?branch=master

Really though, the easiest way is to simply create a branch on the repo with these changes. I've done that [here](https://github.com/Azure/azure-docs-sdk-dotnet/tree/smoke-test-hybridrowexclude) When the publishing build finishes, we'll be able to review the result.